third_party/terraform/utils/utils.go.erb

```diff
@@ -477,3 +477,16 @@ func absoluteDomainSuppress(k, old, new string, _ *schema.ResourceData) bool {
 	return old == new
 }
 <% end -%>
+
+func getInterconnectAttachmentLink(config *Config, project, region, ic string) (string, error) {
+	if !strings.HasPrefix(ic, "https://www.googleapis.com/compute/") {
```

Comment:
You should not be string matching for this full URL because it will break for beta resources or when `www.googleapis.com/compute` eventually moves to `compute.googleapis.com`. If you want to check if a link vs a `name` you could build a more generic method such as isLinkOrName and check for the existence of `/`'s. AFAIK no resource names are allowed to have slashes in them.

Reply:
[Here's](https://cloud.google.com/logging/docs/reference/v2/rest/v2/projects.metrics#LogMetric) a resource that can contain slashes in its name. I'll just check for slashes in this function specifically - and update `getVpnTunnelLink` to do likewise. Fair?

Reply:
lol touche :) yeah that's fine.